pygmt/modules.py

```diff
@@ -11,14 +11,26 @@
     data_kind,
     dummy_context,
     fmt_docstring,
+    kwargs_to_strings,
     use_alias,
 )
 
 
 @fmt_docstring
-@use_alias(V="verbose")
+@use_alias(
+    C="per_column",
+    D="tiles",
+    F="geographic",
+    I="spacing",
+    T="nearest_multiple",
+    L="force_scan",
+    M="minmax_pos",
+    R="region",
+    V="verbose",
+)
+@kwargs_to_strings(R="sequence")
```

Comment:
It would be better to support `tiles = [xoff, yoff]` and `spacing = [dx, dy]`.
```suggestion
@kwargs_to_strings(R="sequence", D="sequence", I="sequence")
```